### PR TITLE
Document hot issues in release process

### PR DESCRIPTION
**Pull request by roycaihw**
Two hot issues that require manual fixes during release processes. Potentially these fixes can be automated.

/assign @yliaog 
/cc @palnabarun @scottilee

### CODE REVIEW

File: devel/release.md [hunk 1]
```diff
@@ -85,9 +85,22 @@ scripts/update-client.sh
 and commit changes (should be only version number changes) to the release branch.
 Name the commit something like "Update version constants for XXX release".
 
-```bash
-git push upstream $RELEASE_BRANCH
-```
+## Hot issues
```
Comment (yliaog) [inline devel/release.md]: could you move it to the top? so people will see it clearly

Comment (roycaihw) [reply devel/release.md]: done

File: devel/release.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ The release process of the python client involves creating (or updating) a relea
...
+***The steps in this section should be performed after you finished the section "Update release tags".***
+
+1. Restore custom object patch hehavior. You should apply [this commit](https://github.com/kubernetes-client/python/pull/995/commits/9959273625b999ae9a8f0679c4def2ee7d699ede)
+to ensure custom object patch behavior is backwards compatible. For more
+details, see [#866](https://github.com/kubernetes-client/python/issues/866) and
+[#959](https://github.com/kubernetes-client/python/pull/959).
+
+2. Add alias package kubernetes.client.apis with deprecation warning. You need
+to add [this file](https://github.com/kubernetes-client/python/blob/0976d59d6ff206f2f428cabc7a6b7b1144843b2a/kubernetes/client/apis/__init__.py)
+under `kubernetes/client/apis/` to ensure the package is backwards compatible.
```
Comment (micw523) [inline devel/release.md]: Can we ask them to cherry-pick the commit instead or it was multiple commits?

Comment (roycaihw) [reply devel/release.md]: It could be done either by cherry-picking the commit or by copy-pasting the file.

**Pull request merged**